### PR TITLE
Film review parsing

### PR DESCRIPTION
**Pull request by tomrf1**
Uses OMDB to get some extra info.
I'll implement the atom stuff in the next PR.

### CODE REVIEW

File: films/src/test/scala/com/gu/film_review_parser/FilmReviewParserSpec.scala [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import com.gu.contentapi.client.model.v1.Content
+import com.gu.contentapi.json.CirceDecoders._
+import com.gu.film_review_parser.parsers.FilmReviewParser
+import org.scalatest.{FunSuite, Matchers}
+import utils.JsonHelpers
+
+class FilmReviewParserSpec extends FunSuite with Matchers {
+  test("parse a Bad Santa 2 review") {
+    val farCry = JsonHelpers.decodeFromFile[Content]("films/src/test/resources/bad-santa-2.json")
```
Comment (LATaylor-guardian) [inline films/src/test/scala/com/gu/film_review_parser/FilmReviewParserSpec.scala]: Copy and paste error.

**Comment by LATaylor-guardian** [comment]
Nice work! 👍

File: films/src/main/scala/com/gu/film_review_parser/ParsedFilmReview.scala [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+                            reviewSnippet: String,
+                            title: String,
+                            genre: String,
+                            year: String,
+                            imdbId: String,
+                            directors: List[String],
+                            actors: List[String]
+                           )
+
+object ParsedFilmReview {
```
Comment (tomrf1) [inline films/src/main/scala/com/gu/film_review_parser/ParsedFilmReview.scala]: the conversion to atoms will happen in here

File: build.sbt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 
 val AwsSdkVersion = "1.11.46"
-
+val CirceVersion = "0.5.0-M2"
```
Comment (mchv) [inline build.sbt]: circe `0.6` has been released!

Comment (mchv) [inline build.sbt]: @tomrf1 make sense!

**Pull request merged**